contact-duplicate/src/app/extensions/package.json

```diff
@@ -6,5 +6,18 @@
   "dependencies": {
     "@hubspot/ui-extensions": "latest",
     "react": "^18.2.0"
+  },
+  "scripts": {
+    "lint": "eslint . --ext .tsx,.ts",
+    "lint:fix": "eslint . --ext .tsx,.ts --fix",
+    "typecheck": "tsc --noEmit --jsx react-jsx --esModuleInterop --lib WebWorker --lib esnext --skipLibCheck .**/*.ts[x]",
```

Comment:
@theJoestJoeToEverJoe do you know any shortcut for reliably running the type checker that's compatible with the vite TS settings/defaults?